assets/locales/en/segment-info.json

```diff
@@ -200,6 +201,10 @@
     "waiting-area|sparse": "Someone",
     "waiting-area|empty": "No one",
     "flex-type|rideshare": "Rideshare",
-    "flex-type|taxi": "Taxi"
+    "flex-type|taxi": "Taxi",
+    "occupants|empty": "Empty",
+    "occupants|full": "Full",
+    "elevation|sidewalk": "Sidewalk level",
+    "elevation|road": "Street levelx"
```

Comment:
I think the `x` at the end might be a typo?

Reply:
Indeed it is, thanks for catching it!